cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts

```diff
@@ -83,7 +88,19 @@ const assetObject3 = [
     c8y_IsDeviceGroup: {}
   }
 ];
-const roomProperties = [{ label: 'Color', isRequired: 'false' }];
+const roomProperties = [{ label: 'Color', isRequired: 'false' }, { label: 'Location', isRequired: false }];
+const marker = "div[class*='dlt-c8y-icon-marker']"; 
+const mapZoomIn = "a[title='Zoom in']";
+const mapZoomOut = "a[title='Zoom out']";
+const mapFullScreen = "c8y-asset-location button[title='Full screen']";
+const moveBars = 'c8y-asset-property-selector td > i';
```

Comment:
moveBars is not descriptive enough. Please rename it to either propertyDragHandle or something meaningful

Reply:
Renamed it to propertyDragHandle